test/units/module_utils/net_tools/test_netbox.py

```diff
@@ -15,7 +15,7 @@
 
 def test_normalize_data():
     assert "name" not in QUERY_TYPES
-    assert QUERY_TYPES.get("rack") == "slug"
+    assert QUERY_TYPES.get("rack") == "name"
```

Comment:
You'll have to change L27 to match the `name` of the rack. ex. `Some rack` so the test passes for it.